### DISCUSSION TITLE
Group associations docs

### DISCUSSION THREAD

**Discussion by zeev-mindali** [zwave-js/zwave-js-ui#3455] [Q&A] [answered]
Dear all,

where i can read about the assosiations? what is report / lifeline and so on....

**Comment by robertsLando** [comment, marked as answer]
https://www.silabs.com/documents/public/application-notes/APL12956-Z-Wave-Association-Basics.pdf

Basically Lifeline is the association with the main controller and is automatically setup by zwave-js driver on node inclusion, that is needed in order to send updates to controller, without that node will never send anything to controller
